src/com/visualization/piechart/piechart.js

```diff
@@ -0,0 +1,69 @@
+import {h, Component} 				from 'preact/preact';
+
+import PieSegment					from 'com/visualization/piechart/piesegment';
+
+export default class PieChart extends Component {
+
+	constructor( props ) {
+		super(props);
+
+    }
+
+    // Converts an array of number to an array of percentages
+    convertToPercentage ( values ) {
```

Comment:
could be static or own function outside of class for easier reuse

Reply:
When it's something this small, I prefer to wait until I actually need something a 2nd time before relocating. If we had a 2nd chart type, different story. Right now it's just the one.

Reply:
If i add more chart types i'll break this (and probably all the legend stuff) out into different files that can be used by all of the charts/graphs ect.

Reply:
to me this is a maths helper function that has very little to do with the graphs and could be in separate file that doesn't deal with components and you import when needed. 

bad name would be `helpers.js`  but it explains the purpose. (To me that's where the `cN` function belongs too rather than the global state.)